CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 9.2.0 "metal-man" (2020-03-25)
+
```

Comment:
I would add a note here about supporting TypeScript 3.8

Reply:
Added a note to say that the release supports 3.8